### ISSUE TITLE
ADD SUPPORT TO IMPORT  PLAYLIST.JSON 

### ISSUE THREAD

**Issue by Devsathome** [HemantKArya/BloomeeTunes#80]
Support for json file import i have some file from blackhole playlists that in .Json file extension , boolmee and blackhole music having code base on flutter dart is it possible to  import. Json file, the playlist contain only song from saavan and yt
![IMG_20241212_201638](https://github.com/user-attachments/assets/221cb9d9-c781-4604-838d-d6d698f62787)

https://github.com/user-attachments/assets/ec34a1df-dae9-4d82-ab01-5ac8c77196ae

**Comment by HemantKArya** [comment]
Yeah was thinking about this feature to support importing files from different apps but it will take time and read how others save music in json style. so, for now, I will leave this as open contributing option. so, if anyone wants to contribute this then please suggest it here and if I will also look into this in future.
